Source/Tools/FEXRootFSFetcher/Main.cpp

```diff
@@ -1261,41 +1309,56 @@ int main(int argc, char **argv, char **const envp) {
             }
             else {
               Args.pop_back();
-              Result = AskForConfirmationList("Squashfuse doesn't work. Do you wish to extract the squashfs file?", Args);
+              Result = AskForConfirmationList(Strings.Strings[1], Args);
               if (Result == 0) {
                 UseImageAs = ArgOptions::CompressedImageOption::OPTION_EXTRACT;
               }
             }
           }
           else {
-            if (WorkingAppsTester::Has_Squashfuse) {
+            if (HasMounter) {
               Args.erase(Args.begin());
-              Result = AskForConfirmationList("Unsquashfs doesn't work. Do you want to use the squashfs file as-is?", Args);
+              Result = AskForConfirmationList(Strings.Strings[2], Args);
               if (Result == 0) {
                 // We removed an argument, Just change "As-Is" from 0 to 1 for later logic to work
                 UseImageAs = ArgOptions::CompressedImageOption::OPTION_ASIS;
               }
             }
             else {
               Args.erase(Args.begin());
-              ExecWithInfo("Unsquashfs and squashfuse isn't working. Leaving rootfs as-is");
+              ExecWithInfo(Strings.Strings[3]);
               UseImageAs = ArgOptions::CompressedImageOption::OPTION_ASIS;
             }
           }
         }
 
         if (UseImageAs == ArgOptions::CompressedImageOption::OPTION_EXTRACT) {
           auto FolderName = filename.substr(0, filename.find_last_of('.'));
-          if (UnSquash::UnsquashRootFS(RootFS, PathName, FolderName)) {
+          if (ExtractHelper(RootFS, PathName, FolderName)) {
             // Remove the .sqsh suffix since we extracted to that
```

Comment:
Should either add the erofs extension here or rephrase the comment to make it agnostic of the image type.

Reply:
Updated comment.